### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by oandalib**
Thanks for the great work!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,8 +257,8 @@ Second, you can configure the access credentials via the Rails configuration obj
 
 ```ruby
 # Set authentication credentials for Litestream
-config.litestream.username = Rails.application.credentials.litestream.username
-config.litestream.password = Rails.application.credentials.litestream.password
+config.litestream.username = Rails.application.credentials.dig(:litestream, :username)
```
Comment (fractaledmind) [inline README.md:260]: Why this change? The dot syntax is supported by Rails for a few point releases

**Review by fractaledmind — APPROVED** [review]
This is great. Thanks for these fixes

File: README.md (hunk 1 above)
Comment (oandalib) [reply README.md:260]: I was using nested credentials for an unrelated purpose in Rails 8, and I ran into issues during Kamal/Docker deployment (assets:precompile). Using dig solved the issue.
